Dockerfile

```diff
@@ -119,11 +122,17 @@ COPY --from=builder /static ${APP}/static
 
 WORKDIR ${APP}
 
+# Create the user
+RUN if [[ "${USER_UID}" != "0" ]]; then (groupadd --gid $USER_GID $USERNAME \
+    && useradd --uid $USER_UID --gid $USER_GID -m $USERNAME \
+    && chown -R $USER_GID:$USER_GID ${APP}); fi
```

Comment:
When all is stuffed together anyway, maybe we can do this instead:

```suggestion
# Create the user
RUN [[ "${USER_UID}" != "0" ]] \
    && groupadd --gid $USER_GID $USERNAME \
    && useradd --uid $USER_UID --gid $USER_GID -m $USERNAME \
    && chown -R $USER_GID:$USER_GID ${APP}
```

Reply:
Wouldn't this fail the build for USER_UID=0 (root user) like this?

Reply:
You're right. A `|| true` suffix could fix that. But that would be ugly.